src/style.css

```diff
@@ -1,3 +1,94 @@
+html {
+  font-size: 12px;
+}
+
 body {
   margin: 0;
+  padding: 0;
+  box-sizing: border-box;
```

Comment:
move to `*` selector

Reply:
Not resolved

Reply:
If I put this style box-sizing: border-box;
 in *{}, it stops working, please tell me if I should move this style to *{}. ?